src/app/frontend/deploy/deployfromsettings.scss

```diff
@@ -26,7 +39,6 @@ md-progress-linear {
   }
 }
 
-.kd-deploy-input-row {
-  margin-bottom: $baseline-grid;
-  margin-top: $baseline-grid;
+.progress-or-message-container {
```

Comment:
Ha! yes, I see it's buttery smooth on Chrome :) i see the glitch in Firefox where the progress bar appears about 5 px below the input line (I've not managed to change this behaviour - but I did find this which has clearly been fixed for Chrome: https://github.com/angular/material/issues/2540)